assets/video-section.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .video-section__media.deferred-media {
-  box-shadow: var(--media-shadow-horizontal-offset) var(--media-shadow-vertical-offset) var(--media-shadow-blur-radius) rgba(var(--color-foreground), var(--media-shadow-opacity));
+  box-shadow: var(--media-shadow-horizontal-offset) var(--media-shadow-vertical-offset) var(--media-shadow-blur-radius) rgba(var(--color-shadow), var(--media-shadow-opacity));
 }
```

Comment:
The shadow doesn't seem to work on `accent-1` and `inverse`

https://screenshot.click/20-25-98oj1-cds2z.png

Reply:
It's expected. The shadow is using the general text color set. So if you're setting one of those background to be the same color than the text, you won't see it. 

Here is the original issue to change the color source: https://github.com/Shopify/dawn/issues/1026

Reply:
Yeah exactly where I was going to answer, I think this was overlooked, for `Inverse` style, we could have an exception to pick `rgba(var(--color-foreground)`. But I need to think further about it.

Reply:
That would be just in the scenario of a component that can receive shadow is on top of another container that can receive color scheme though.